### ISSUE TITLE
Scheduler

### ISSUE THREAD

**Issue by IceFlovver** [amosproj/amos2022ws01-firmware-scraper#20]
## User story
1. As a user
2. I want to be able to type in the time interval
3. So that the software will start the action at the set time point

## Acceptance criteria
* it is possible to input time
* the software runs at the set time period

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
